myTrip/mytrip/settings.py

```diff
@@ -62,13 +64,13 @@
 
 LANGUAGE_CODE = 'en-us'
 
-TIME_ZONE = 'UTC'
+TIME_ZONE = 'Europe/Kiev'
 
 USE_I18N = True
 
 USE_L10N = True
 
-USE_TZ = True
+USE_TZ = False
```

Comment:
had to change it to False cause of conflict with datetime module.
when set to True throws this while testing:
``` RuntimeWarning: DateTimeField received a naive datetime ```